scrapi/base/transformer.py

```diff
@@ -39,16 +39,19 @@ def _transform(self, schema, doc):
         return transformed
 
     def _transform_iterable(self, l, doc):
-        docs = []
 
         if isinstance(l[0], tuple) and len(l) == 2:
             return self._transform_args_kwargs(l, doc)
 
-        for value in l:
+        fn, values = l[-1], l[:-1]
+        args = []
+
+        for value in values:
             if isinstance(value, basestring):
-                docs.append(self._transform_string(value, doc))
+                args.append(self._transform_string(value, doc))
             elif callable(value):
-                return value(*[res for res in docs])
+                args.append(value(doc))
```

Comment:
yes, the new behavior is better. With the old behavior, it would return if it found a function, so you could only ever include a single function. Now you can include a list of functions that can be called on docs, and the last function will be called with the results of the previous functions. It is now more consistent with how the strings are processed, and I think it adds a lot of flexibility. It should cause errors anywhere else in the code base, because the functions were the last in the list/tuple anyway.